.gitignore

```diff
@@ -14,6 +14,7 @@ cscope.*
 venv
 .venv
 .idea
+.DS_Store
 
```

Comment:
*note:* https://github.com/coderbyheart/first-principles/issues/30

Reply:
Thanks, I didn't know about global gitignore. Reverted the change in #278.

Reply:
🚀